checkov/terraform/checks/resource/azure/AppServiceMinTLSVersion.py

```diff
@@ -1,10 +1,12 @@
+from typing import Any, List
+
 from checkov.common.models.enums import CheckResult, CheckCategories
 from checkov.terraform.checks.resource.base_resource_value_check import BaseResourceValueCheck
 
 
 class AppServiceMinTLSVersion(BaseResourceValueCheck):
     def __init__(self):
-        name = "Ensure web app is using the latest version of TLS encryption"
+        name = "Ensure web app uses at least TLS 1.2"
```

Comment:
```suggestion
        name = "Ensure web app is using the latest version of TLS encryption"
```
You can keep the name. Changing the name has broader implications and TLS 1.2 is still considered an acceptable version.

Reply:
Thanks. Fixed in [70a8eaa](https://github.com/bridgecrewio/checkov/pull/6956/commits/70a8eaaad1754050a139710a54b03b8a1b05da64).